NATIVE_NOTES.md

```diff
@@ -0,0 +1,110 @@
+# IMPORTANT
```

Comment:
i just put this doc here since it was easy, but it might be better in notion or google docs?